src/core/AvatarClient.ts

```diff
@@ -33,6 +35,7 @@ export class AvatarClient extends HTTPClient {
   constructor(config: AvatarClientConfig) {
     super(config.baseUrl ?? 'https://avatar.alpha.school', config.apiKey);
     this.avatarId = config.avatarId;
+    this.landmarks = config.landmarks ?? false;
```

Comment:
suggestion: I wonder if is not better to just have this always enabled if the avatar has landmarks 🤔

Reply:
I'll leave it false for now, so we can first understand if all the avatars will have customization or not, but then we can change it to true.